src/components/ToolsBar/ToolsBar.tsx

```diff
@@ -108,6 +114,15 @@ export function ToolsBar() {
             onChange={handleRadioClick}
           />
         ))}
+        {imageUploadEnabled && (
+          <ToolbarButton
+            aria-label={t('toolsBar.uploadTool', 'Upload image')}
+            {...getRootProps()}
+          >
+            <input {...getInputProps()} onClick={undefined} />
+            <UploadIcon sx={{ height: 22 }} />
```

Comment:
Do we need this explicit: `height: 22`?

Reply:
Yes. It is required to match the designs. That is where the file comes from.
It has to be sized to match into the toolbar.